package.json

```diff
@@ -98,6 +98,26 @@
     "webpack-manifest-plugin": "5.0.0",
     "workbox-webpack-plugin": "4.3.1"
   },
+  "resolutions": {
+    "bnc-onboard/**/axios": "0.27.2",
```

Comment:
1. What does `**` mean?
2. Have you manually edited yarn.lock, or have all updates performed automatically?

Reply:
1. ** means "any number of nested submodules".
2.  Automatically.